pkg/config/config.go

```diff
@@ -40,7 +40,7 @@ func GetKubeConfig() (*rest.Config, error) {
 		return configFromFlags(kubeConfig)
 	}
 
-	return nil, fmt.Errorf("Could not locate a kubeconfig")
```

Comment:
remove?

Reply:
No , the next line is not showing up ? I aded return nil, fmt.Errorf("could not locate a kubeconfig")
+ return nil, fmt.Errorf("could not locate a kubeconfig")